docs/openstack_dataplaneservice.md

```diff
@@ -125,9 +125,10 @@ OpenStackDataPlaneServiceSpec defines the desired state of OpenStackDataPlaneSer
 | configMaps | ConfigMaps list of ConfigMap names to mount as ExtraMounts for the OpenStackAnsibleEE | []string | false |
 | secrets | Secrets list of Secret names to mount as ExtraMounts for the OpenStackAnsibleEE | []string | false |
 | openStackAnsibleEERunnerImage | OpenStackAnsibleEERunnerImage image to use as the ansibleEE runner image | string | false |
-| tlsCertsEnabled | TLSCertsEnabled - Whether the nodes have TLS certs | *bool | false |
+| tlsCertsEnabled | TLSCertsEnabled - Whether the nodes have TLS certs | bool | true |
```

Comment:
Why does this show the default as true? I do see you have default as false where the field is actually defined? Do the docs need to be regenerated?

Same for tlsEnabled.

Reply:
The final column here is not the default value, but rather whether or not the field is required.

You'll notice I changed from a pointer to a boolean to plain boolean with a default value (which is set to false)

Reply:
ok, that makes sense.